src/lib.rs

```diff
@@ -110,6 +110,8 @@ pub mod stdlib;
 pub type Positive = std::num::NonZeroU32;
 /// Non-negative integer.
 pub type NonNegative = u32;
+/// Natural integer as defined by VHDL.
+pub type Natural = NonNegative;
```

Comment:
Maybe we should move this type def to the vhdl generator module. What do you think?